apps/studio/components/interfaces/Home/AdvisorWidget.tsx

```diff
@@ -0,0 +1,328 @@
+                  performanceWarningCount,
+                  isLoadingLints
+                )}
+              </TabsContent>
+            </CardContent>
+          </Tabs>
+        </Card>
+
+        <Card className="h-80 flex flex-col">
+          <CardHeader className="h-10 flex-row items-center justify-between py-0 pl-4 pr-2">
+            <CardTitle>Slow Queries</CardTitle>
+            <ButtonTooltip
+              asChild
+              type="text"
+              className="!mt-0 w-7"
+              icon={<ExternalLink />}
+              tooltip={{
+                content: {
+                  side: 'bottom',
+                  text: `Open Query Performance Advisor`,
+                },
+              }}
+            >
+              <Link href={`/project/${projectRef}/advisors/query-performance`} />
+            </ButtonTooltip>
+          </CardHeader>
+          <CardContent className="!p-0 flex-1 overflow-y-auto">
+            {isLoadingSlowestQueries ? (
+              <div className="space-y-2 p-4">
+                <ShimmeringLoader />
+                <ShimmeringLoader className="w-3/4" />
+                <ShimmeringLoader className="w-1/2" />
+                <ShimmeringLoader className="w-3/4" />
+                <ShimmeringLoader className="w-1/2" />
+              </div>
+            ) : top5SlowestQueries.length === 0 ? (
+              <div className="flex-1 flex flex-col h-full items-center justify-center gap-2">
+                <Activity strokeWidth={1.5} size={20} className="text-foreground-muted" />
+                <p className="text-sm text-foreground-light">
+                  No slow queries found in the selected period
+                </p>
+              </div>
+            ) : (
+              <Table className="text-xs font-mono max-w-full">
+                <TableHeader>
+                  <TableRow>
+                    <TableHead className="text-foreground-lighter truncate py-2 h-auto">
+                      Query
+                    </TableHead>
+                    <TableHead className="text-foreground-lighter truncate py-2 h-auto">
+                      Avg time
+                    </TableHead>
+                    <TableHead className="text-foreground-lighter truncate py-2 h-auto">
+                      Calls
+                    </TableHead>
+                  </TableRow>
+                </TableHeader>
+                <TableBody>
+                  {/* Added explicit types for map parameters */}
+                  {top5SlowestQueries.map((query: SlowQuery, i: number) => (
+                    <TableRow key={i}>
+                      <TableCell className="font-mono truncate max-w-xs py-2">
+                        {query.query}
+                      </TableCell>
+
+                      <TableCell className="font-mono truncate max-w-xs py-2">
+                        {typeof query.avg_time === 'number'
+                          ? `${query.avg_time.toFixed(2)}ms`
```

Comment:
opting to change this to seconds actually - there was some feedback previously that it was easier to glance in seconds as the numbers for ms were too large haha